.jpb/persistence-units.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="PersistenceUnitSettings">
+    <persistence-units>
+      <persistence-unit name="Default">
+        <packages>
+          <package value="" />
+        </packages>
+      </persistence-unit>
+    </persistence-units>
+  </component>
+</project>
```

Comment:
```suggestion
</project>

```

Reply:
Can you explain why this xml file is added?